MicrosoftFluentUI.podspec

```diff
@@ -281,7 +281,7 @@ fi', :execution_position => :before_compile }
     shimmer_ios.platform = :ios
     shimmer_ios.dependency 'MicrosoftFluentUI/Core_ios'
     shimmer_ios.dependency 'MicrosoftFluentUI/Utilities_ios'
-    shimmer_ios.source_files = ["ios/FluentUI/Shimmer/**/*.{swift,h}"]
+    shimmer_ios.source_files = ["ios/FluentUI/Vnext/Shimmer/**/*.{swift,h}"]
```

Comment:
@rdeassis is this still something we want to do? What's the value of having these controls live in a different folder, especially if there's only one version of the control in the project at a time?